project4/3700crawler

```diff
@@ -307,7 +439,13 @@ if __name__ == '__main__':
     args = parser.parse_args()
 
     # Create a WebCrawler with the given username and password
-    crawler = WebCrawler(args.username, args.password)
+    crawler = HydraCrawler(args.username, args.password)
+
+    logging.getLogger().setLevel(logging.ERROR)
 
     # Start crawling
-    crawl(crawler)
+    start_time = time.time()
+    keys = crawler.crawl()
+    print('Runtime {0:.2f} minutes'.format((time.time() - start_time)/60.0))
```

Comment:
Need to get rid of or comment out this line because the assignment says to only print the secret flags